packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -33,8 +40,15 @@ class Checkbox extends StatelessWidget {
     this.onChanged
   }) : super(key: key);
 
+  /// Whether this checkbox is checked or unchecked.
   final bool value;
+
+  /// The color to use when this checkbox is checked.
+  ///
+  /// Defaults to accent color of the current theme.
   final Color activeColor;
+
+  /// Called when the user checks or unchecks the checkbox.
```

Comment:
This should talk about the fact that you have to update the value and pass it back in, probably.

Also that if this is null, the widget will appear disabled.